core_api_c_binding/src/lib.rs

```diff
@@ -27,17 +27,21 @@ impl Logger for NullLogger {
 pub unsafe extern "C" fn holochain_new(ptr: *mut Dna) -> *mut Holochain {
     let agent = Agent::from("c_bob".to_string());
 
-    let context = Arc::new(Context::new(
+    let context = Context::new(
         agent,
         Arc::new(Mutex::new(NullLogger {})),
         Arc::new(Mutex::new(SimplePersister::new())),
-    ));
+        "holostorage",
```

Comment:
What is this "holostorage" string? What does it do? 

I think it might not be a good choice of name, since it once again sets up a holo <> holochain confusion

Reply:
Changes still incoming :) have a push that uses the static storage variable instead.

Reply:
Nice, sorry :p

Reply:
hehe no problem!! For this to work such questions need to be asked hehe